packages/jest-snapshot/src/__tests__/printSnapshot.test.ts

```diff
@@ -803,14 +827,16 @@ describe('printSnapshotAndReceived', () => {
       serialize(received),
       received,
       expand,
+      {},
     );
 
   // Simulate custom raw string serialization.
   const testWithoutStringify = (
     expected: string,
     received: string,
     expand: boolean,
-  ): string => printSnapshotAndReceived(expected, received, received, expand);
+  ): string =>
+    printSnapshotAndReceived(expected, received, received, expand, {});
```

Comment:
Could also make sense to mark `snapshotFormat` as optional here:

https://github.com/facebook/jest/blob/9984ba12facddf3170ebc73ac31765ca50dcf343/packages/jest-snapshot/src/printSnapshot.ts#L233-L239

Reply:
sure